### ISSUE TITLE
Incorrect Portuguese translation of the 'second' keyword in languages.yaml

### ISSUE THREAD

**Issue by VOIDCRUSHER** [scrapinghub/dateparser#113]
dateparser v0.3.0 on Ubuntu 14.04

The languages.yaml file has the incorrect English to Portuguese Translation of 'second.'  
'segunda' is the plural adjective form of second as an ordered position where as 'segundo' is the desired term for the unit of time. The bug manifests itself as follows:

```python
>>> parse(u'1 segundo atrás')
>>>
>>> parse(u'1 segunda atrás')
datetime.datetime(2015, 7, 13, 9, 19, 43, 484810)
```

**Comment by asadurski** [comment]
Thank you for reporting this problem, @VOIDCRUSHER. Please take a look at PR https://github.com/scrapinghub/dateparser/pull/114 which is aimed at fixing it. Your feedback is very appreciated.

**Comment by waqasshabbir** [comment]
Fixed in #114.
